examples/cactus-workshop-examples/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@hyperledger/cactus-workshop-examples",
+  "version": "1.0.0",
```

Comment:
@RafaelAPB Please when the time comes bump this version to whatever the next release will be at the time.